integration-tests/src/test/resources/application.properties

```diff
@@ -1 +1,2 @@
-quarkus.log.file.enable=true
\ No newline at end of file
+quarkus.log.file.enable=true
+%test.quarkus.log.console.json.pretty-print=true
```

Comment:
This is not default config, and changes behavior for all existing test. I don't think pretty print should be on in the integration-tests. Should be enough to test with normal formatter test.

Reply:
Good spot.

Removed. It was not needed and possibly residue from my efforts last year.